sdk/keyvault/keyvault-certificates/test/public/CRUD.spec.ts

```diff
@@ -83,7 +87,9 @@ describe("Certificates client - create, read, update and delete", () => {
 
   // On playback mode, the tests happen too fast for the timeout to work - in browsers
   it("can create a certificate with requestOptions timeout", async function (this: Context) {
-    recorder.skip("browser", "Timeout tests don't work on playback mode.");
+    if (!isNode && isPlaybackMode()) {
+      this.skip();
```

Comment:
Can you unskip this?
I believe there shouldn't be a problem with playback with the new recorder in place.

Reply:
Same for other skipped tests.

Reply:
Cool, I unskipped the ones which are skipped for this reason and they are passing fine. There are a few which are skipped due to a dependency on `fs` which of course still need to be skipped in browser.